### ISSUE TITLE
Integrate morango garbage collection

### ISSUE THREAD

**Issue by jamalex** [learningequality/kolibri#8238]
Depends on: https://github.com/learningequality/morango/issues/184

## Background
We added a [garbage collection command in morango](https://github.com/learningequality/morango/pull/120/files) for clearing out old stale syncsessions, but need to run it somewhere in Kolibri, ideally just before we do the vacuum jobs.

## Deliverables
A new management command of the same name `cleanupsyncs` should:
- exist to override Morango's
- check if Kolibri is unprovisioned and raise an error if so, to prevent confusion when running this command manually from the CLI without the proper Kolibri home
- execute Morango's command if provisioned

A new task should be created which should:
- invoke morango's `cleanupsyncs` command with passed in parameters

Add a `FacilityDataSyncHook` which should:
- enqueue the above task when the sync completes successfully, passing the sync filter to the command's matching parameter and either:
  - the client instance ID to the command's matching parameter when the device is the client WRT the sync, or
  - the server instance ID to the command's matching parameter when the device is the server WRT the sync

## Future considerations
Note that that command only actually deletes buffer records, and just _closes_ sync/transfer sessions. We may want to delete older / unnecessary syncsession and transfersession records as well, as they will start to pile up more quickly now with the automated syncing.
